src/components/Feeds/BusinessFeed.js

```diff
@@ -24,7 +26,17 @@ function BusinessFeed({ businesses, onSearch, selectedFilters, loadingState }) {
         selectedFilters={selectedFilters}
         isSearching={loadingState === LOADING_STATE.SEARCHING}
       />
-      {!initialLoad && hasResults && (
+      {(searching || initialLoad) && (
+        <Box mb={10}>
+          <SimpleGrid columns={[null, 1, 2]} spacing={10}>
+            {[...Array.from(new Array(4))].map((val, index) => (
+              <Skeleton key={index} height="300px" />
```

Comment:
I believe that we can be a little bit more tricky with the `<Skeleton>` component here to better match the width and height of the incoming cards!

As per the documentation here: https://chakra-ui.com/skeleton I believe that we can actually wrap the `<ResultCard>` with a `<Skeleton>` component in order to have it take on the width and height of the incoming card. Did you play around with this by chance? It's entirely possible that this may not be the right solution, just wanted to point out the possibility!

Reply:
I actually tried it. In our case, we have an image in the card which means that `<ResultCard>`, being a child of `<Skeleton>`,  should take a category for it to display an image skeleton, otherwise it will have no image and we will not get the desired height and width of the skeleton cards. What I can do here is the following:
1. Provide a category to `<ResultCard>` , for example `{ id: 1, name: 'Entertainment' }` which will make it take an image and thus showing a skeleton loader for the image.
2. The component which affects the width of the card and makes it take the width it currently does is  `<Heading>` in `<ResultCard>`. This is why I gave it a name, as shown below, which is long enough to make the 2 grid elements on the same row take the full width of the page which is `1220px`. 
3. I was observing a weird bug when providing `<ResultCard>`, as a child of `<Skeleton>`,  with a `websiteUrl` prop. The two anchors which show when given that prop were having a constant width which gave the page a horizontal scroll when resizing. Not providing the prop solved the problem. I am not sure exactly what caused it. 

Below is how the new loading skeleton might look like. 
```
 <SimpleGrid columns={[null, 1, 2]} spacing={10}>
            {[...Array.from(new Array(4))].map((_, index) => (
              <Skeleton key={index}>
                <ResultCard
                  category={{ id: 1, name: 'Entertainment' }}
                  name="Saving Small Businesses In Chicago- Skyway Bowl"
                  description="PlaceHolder Description"
                  location="Placeholder Location"
                  donationUrl="Placeholder Url"
                />
              </Skeleton>
            ))}
          </SimpleGrid>
```

Reply:
Yeah let's give that a shot if we can! Totally understandable if the amount of dynamic content fed to these cards is a bit too much to reliably get the inherited width and height. Worst case scenario we can set a `width` of 100%, a max width of whatever the value is for a card on desktop, and a height. 

Appreciate your thoroughness here!

Reply:
How about we give the parent `<Box>` component in  `<BusinessFeed>`  a `width` of `100%` ? It currently has a `maxWidth` of `1220px`. This way we do not give the skeleton cards any width, since their parent will take the entire width which is the same as its parent (in this case `1220px`). As for the height, I believe we cannot exactly get the exact value even if we set it to `100%` since the content is dynamically generated. I a more than happy to discuss further.

Reply:
Totally fine with that approach! If we can just get the skeleton cards to match the size as close as we can I'll be happy! Appreciate the discussion!!